Clients/src/presentation/components/ProjectCard/styles.ts

```diff
@@ -1,36 +1,43 @@
+import light from "../../themes/light";
```

Comment:
@popovaevgeniya 
Thank you for replacing the theme value. 🙏🏻
However, this is the wrong way of using the theme.

`const theme = useTheme();` from **MUI package**

In things like `light.shape...` or `light.palette...`, `light.` must be replaced with `theme.`